packages/builder/cypress/integration/templates/HR/jobApplicationTracker.spec.js

```diff
@@ -35,19 +35,10 @@ filterTests(["all"], () => {
             cy.get(".spectrum-Button").contains("Create app").click({ force: true })
         })
 
-        // Publish App
+        // Publish App & Verify it opened
         cy.wait(2000) // Wait for app to generate
-        cy.get(".toprightnav").contains("Publish").click({ force: true })
-        cy.get(".spectrum-Dialog-grid").within(() => {
-            cy.get(".spectrum-Button").contains("Publish").click({ force: true })
-        })
-
-        // Verify Published app
-        cy.wait(2000) // Wait for App to publish and modal to appear
-        cy.get(".spectrum-Dialog-grid").within(() => {
-            cy.get(".spectrum-Button").contains("View App").click({ force: true })
-            cy.window().its('open').should('be.calledOnce')
-        })
+        cy.publishApp(true)
```

Comment:
Is it a bit redundant to do publish tests here? As long as the app can be created I think that's more than enough, the publish shouldn't be affect by the import source

Reply:
Yeah I can understand where you are coming from. In terms of these templates we should just ensure the app is being created. The idea was to ensure that clicking 'View app' after publishing opened the app in a new tab. I will instead have one test that does this for a generic app and refactor those template tests